packages/artillery/lib/platform/aws-ecs/legacy/bom.js

```diff
@@ -34,7 +35,8 @@ function createBOM(absoluteScriptPath, extraFiles, opts, callback) {
           opts: {
             scriptData,
             absoluteScriptPath,
-            flags: opts.flags
+            flags: opts.flags,
+            scenarioPath: opts.scenarioPath // Absolute path to the file that holds scenarios
```

Comment:
Adding the `scenarioPath` to options here so we can access and pass it to `resolveConfigTemplates` making it available in config for distributed runs.

Both Lambda and Fargate use this function and will pass the path to scenarios to `opts` from their side